test/unit_tests/apps/owning-a-home/js/explore-rates/rate-checker-spec.js

```diff
@@ -177,8 +178,9 @@ describe( 'explore-rates/rate-checker', () => {
     beforeEach( () => {
       document.body.innerHTML = HTML_SNIPPET;
       downPaymentDom = document.querySelector( '#down-payment' );
-      rateStructureDom = document.querySelector( '#rate-structure' );
-      armTypeDom = document.querySelector( '#arm-type' );
+
+      /* rateStructureDom = document.querySelector( '#rate-structure' );
+         armTypeDom = document.querySelector( '#arm-type' ); */
```

Comment:
Yes, they were commented out because they were not used and probably reminders that more tests are needed.
Instead of deleting them, I made an attempt to check value of rate-structure and ARM type.
@anselmbradford What is the command line that generates code coverage?